app/views/users/profile-details.njk

```diff
@@ -0,0 +1,109 @@
+{% extends 'layout.njk' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+
+{% block breadcrumbs %}
+  {{
+    govukBackLink({
+      text: 'Back',
+      href: '/'
+    })
+  }}
+{% endblock %}
```

Comment:
I've kept the back link from the original system that just returns the user to the home page, even though that's not technically going "back" as they could have accessed this page from anywhere.

But it still felt the most appropriate way for the user to navigate elsewhere after updating their details.